### DISCUSSION TITLE
fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-local-IPv6/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4456] [Content Help کمک به اصلاح محتوا]
# fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-local-IPv6/

The proxy manager platform.

https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-local-IPv6/

**Comment by miladfff** [comment]
وقتی که روی سرور اصلی میخوام دستور نانو بزنم این ارور رو میده:
ctrl + s
   [ Error writing /etc/netplan/tunnel.yaml: No such file or directory ]

**Reply by miladfff** [reply]
متاسفانه دوبار امتحان کردم نانو ارور میده و سرور به فنا میره و باید دوباره کلا سیستم رو از اون روی سرور ریست شده نصب کرد

**Reply by Fouad-Izak** [reply]
کافیه پکیج نت پلن رو نصب کنید .
apt install netplan.io

**Comment by reza-naghizadeh** [comment]
سلام خسته نباشید 
امکان این هست که بشه کانفینگ هیدیفای رو رویمودم ست کرد؟
